src/mapInteractor.js

```diff
@@ -192,11 +203,35 @@ var mapInteractor = function (args) {
          * (which would propably require detecting keyup events). */
         focusHighlight: true
       },
-
+      /* Set alwaysTouch to false to only add touch support on devices that
+       * report touch support.  Set to true to add touch support on all
+       * devices. */
+      alwaysTouch: false,
       wheelScaleX: 1,
       wheelScaleY: 1,
       zoomScale: 1,
       rotateWheelScale: 6 * Math.PI / 180,
+      /* The minimum angle of rotation (in radians) before the
+       * geo_action.zoomrotate action will allow rotation.  Set to 0 to always
+       * include rotation. */
+      zoomrotateMinimumRotation: 5.0 * Math.PI / 180,
```

Comment:
We haven't camel cased other actions (zoomselect, for instance).  Since this is related to the zoomrotate action, I thought _not_ capitalizing the R made more sense (but I'm hardly convinced of this).